### PR TITLE
Add resourcegroup endpoints

### PR DESCRIPTION
**Pull request by colin014**
Fixes: #790

### CODE REVIEW

File: api/resourcegroup.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+	pkgCommon "github.com/banzaicloud/pipeline/pkg/common"
+	"github.com/gin-gonic/gin"
+	"github.com/sirupsen/logrus"
+)
+
+// GetResourceGroups lists resource groups by secret
+func GetResourceGroups(c *gin.Context) {
+
+	orgID := auth.GetCurrentOrganization(c.Request).ID
+	secretId := c.Query("secretId")
```
Comment (stoader) [inline api/resourcegroup.go]: Shouldn't this be rather a header param then query param? IMHO query params better fit passing in filters.

File: api/resourcegroup.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+		Name: request.Name,
+	})
+
+}
+
+// DeleteResourceGroups deletes resource group by name
+func DeleteResourceGroups(c *gin.Context) {
+
+	orgID := auth.GetCurrentOrganization(c.Request).ID
+	secretId := c.Query("secretId")
```
Comment (stoader) [inline api/resourcegroup.go]: Shouldn't this be rather a header param then query param? IMHO query params better fit passing in filters.

File: api/resourcegroup.go (hunk 1 above)
Comment (colin014) [reply api/resourcegroup.go]: sure, fixed

File: api/resourcegroup.go (hunk 2 above)
Comment (colin014) [reply api/resourcegroup.go]: sure, fix this

**Pull request merged**